httpcore/_async/http11.py

```diff
@@ -17,16 +26,14 @@
 from ..backends.base import AsyncNetworkStream
 from .interfaces import AsyncConnectionInterface
 
-H11Event = Union[
+
```

Comment:
Looking at the continuous integration I think the listing requires this line to be removed.

The `scripts/lint` tool ought to do this for you automatically, or we can remove it as part of this review, through the GitHub interface.

Here's the commit suggestion...

```suggestion
```